classesExamples/WriteAFile.py

```diff
@@ -0,0 +1,6 @@
+my_list = [1, 2, 3]
+my_file = open("firstFile.txt", "w")
+
+for item in my_list:
+    my_file.write(str(item) + "\n")
+my_file.close()
```

Comment:
Sometimes you left two lines at the end of file, sometimes you left none, the standard is to leave just one, I know it sounds like an annoying comment, but this randomness makes your code sloppy, and on a real life environment, someone else will bring it up in the future 🔮

Reply:
I think the standard is to leave a single blank line at the end of a python file: there's also a coding [styleguide](https://www.python.org/dev/peps/pep-0008/), but I think it should be fully configurable using `pylint` along with your editor, so it will warn you and let you know what to change or even fix it automagically